src/frontend/komponenter/Fagsak/Behandlingsresultat/Oppsummeringsboks.tsx

```diff
@@ -361,7 +361,7 @@ const Oppsummeringsboks: React.FunctionComponent<IProps> = ({
                                 onClick={() =>
                                     leggSmåbarnstilleggTilIMåned(småbarnstilleggKorrigering)
                                 }
-                                icon={<AddCircle aria-hidden />}
+                                icon={<PlusCircleIcon aria-hidden />}
```

Comment:
Får ikke så lett til å teste disse, fordi jeg ikke har en sak hvor det er småbarnstillegg. Men disse to ikonene har funket fint over alt ellers, så jeg tenker det burde gå bra :D